apps/sports-helsinki/src/domain/search/searchHeader/searchHeader.module.scss

```diff
@@ -83,33 +86,29 @@
       }
     }
 
-    button:first-child {
-      position: relative;
-
-      &:after {
-        content: '';
-        position: absolute;
-        border: 2px solid black;
-        width: 0;
-        // Add few pixels to make sure border covers the whole height of the container
-        height: calc(100% + 5px);
-        border-left: 0;
-        right: 0;
-
-        @include breakpoints.respond-above(m) {
-          border: none;
-        }
-      }
+    //Select first button element in searchActions div
+    button:first-of-type {
+      border: none;
     }
 
     @include breakpoints.respond-above(m) {
       button:last-child {
-        margin-left: auto;
+        margin-left: 0;
       }
     }
   }
 }
 
+.paddingClosed {
+  padding: $spacing-layout-xs 0 $spacing-layout-xs 0 !important;
```

Comment:
When the search-section is closed, the cap is filled with white (instead of transparent like when it's opened), but it's still unnecessary in my opinion:

<img width="462" alt="image" src="https://user-images.githubusercontent.com/389204/228857679-5ae2f418-46cb-4be0-a098-3320517a990c.png">

```suggestion
```